app/models/vm_or_template/snapshotting.rb

```diff
@@ -16,11 +16,11 @@ module VmOrTemplate::Snapshotting
   end
 
   def newest_snapshot
-    snapshots.sort { |a, b| a.create_time <=> b.create_time }.last
+    snapshots.sort_by(:create_time).last
```

Comment:
I think sort_by.last pattern on a timestamp can be better replaced with `max_by`

```ruby
snapshots.max_by(&:create_time)
```

```ruby
[1] pry(main)> require 'acrive_support/all'
[2] pry(main)> times = 20.times.collect { OpenStruct.new(:timestamp => rand(10.days.ago..Time.now)) }
=> [#<OpenStruct timestamp=2017-02-17 14:19:30 -0500>,
 #<OpenStruct timestamp=2017-02-18 20:14:50 -0500>,
...
 #<OpenStruct timestamp=2017-02-23 13:46:17 -0500>,
 #<OpenStruct timestamp=2017-02-14 02:37:43 -0500>]
[3] pry(main)> times.max_by(&:timestamp)
=> #<OpenStruct timestamp=2017-02-23 13:46:17 -0500>
```